docs/architecture/porting-existing-aspnet-apps/app-startup-differences.md

```diff
@@ -7,8 +7,63 @@ ms.date: 11/13/2020
 
 # App startup differences between ASP.NET MVC and ASP.NET Core
 
+ASP.NET MVC applications lived entirely within Internet Information Server (IIS), the primary web server available on Windows operating systems. Unlike ASP.NET MVC, ASP.NET Core applications are executable applications. You can run them from the command line, using `dotnet run`. They have an entry point method like all C# programs, typically `public static void main()` or a similar variation (perhaps with arguments or async support). This is perhaps the biggest architectural difference between ASP.NET Core and ASP.NET MVC, and is one of several differences that allows ASP.NET Core to run on non-Windows systems.
```

Comment:
I don't see this doc in the [docs preview site](https://review.docs.microsoft.com/en-us/dotnet/architecture/porting-existing-aspnet-apps/introduction?branch=pr-en-us-21160). Where does it show up in the outline? Perhaps I'm just missing it? This is useful content and I want to make sure it's included.

Reply:
@mjrousos sadly I can't get to the docs preview site but hopefully it's there now?

Reply:
Yes, I have access to that. I've been fixing required things but admit I'd left some of the warnings for later. I'll try and knock out the missing descriptions shortly.